lib/trackler/track.rb

```diff
@@ -100,9 +100,9 @@ def img(file_path)
     end
 
     def doc_format
-      default_format = 'md'
-      path = File.join(dir, "docs", "*.*")
-      most_popular_format(path) || default_format
+      TOPICS.map { |topic|
+        DocFile.find(basename: topic.upcase, track_dir: dir, image_path: "").extension
+      }.partition(&:to_s).max_by(&:size).first
```

Comment:
Does this method need to change?

* I think the old version is clearer.
* `EXERCISE_README_INSERT` doesn't get a vote.